fuzz-test/readme.md

```diff
@@ -0,0 +1,20 @@
+# Fuzz test
+Simple setup using AFL or libFuzzer.
+
+## Libfuzzer
+```
+$ mkdir build-libfuzzer
+$ cd build-libfuzzer
+$ cmake ../.. -DCMAKE_C_COMPILER=clang -DCMAKE_C_FLAGS="-fsanitize=fuzzer,address -DLIBFUZZER" -DCMAKE_C_COMPILER_WORKS=On
+$ make fuzz_pet
+$ ./fuzz-test/fuzz_pet
+```
+
+## AFL
+```
+$ mkdir build-afl
+$ cd build-afl
+$ export AFL_USE_ASAN=1
```

Comment:
I see using ASAN is only recommended in specific scenarios: https://afl-1.readthedocs.io/en/latest/notes_for_asan.html

Any reason to add it here?

Reply:
When using AFL and ASAN you might need to use -m none to disable memory limit. At least i had to do that some time ago, but now with AFLPlusPlus i am not sure.

You can just add:
```
cmake ../.. -DCMAKE_C_COMPILER=afl-clang-fast -DCMAKE_C_FLAGS="-m32"
```

Reply:
@sijohans I can't get 32 bit mode to work. I get this error:

```
[-] PROGRAM ABORT : -m32 is not supported by your compiler
         Location : edit_params(), afl-clang-fast.c:652
```

Do you by any chance know what to do here?

Reply:
I didn't manage to get m32 working with afl-clang-fast either. I had to use afl-clang (or afl-gcc).
What system are you using?